packages/stellar-dispatcher/src/index.js

```diff
@@ -0,0 +1,36 @@
+import uuid from 'uuid';
```

Comment:
you should import a sepcific version of uuid v4 probably

Reply:
What?